source/_components/sensor.reddit.markdown

```diff
@@ -30,6 +30,7 @@ To enable this platform, add the following to your `configuration.yaml` file:
   password: !secret reddit_password
   client_id: !secret reddit_client_id
   client_secret: !secret reddit_client_secret
+  sort_by: new
```

Comment:
You can remove this line, only variables that are required in the **first** example config